src/CosmosDB/CosmosDB/help/Get-AzManagedCassandraCluster.md

```diff
@@ -0,0 +1,106 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll-Help.xml
+Module Name: Az.CosmosDB
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cosmosdb/get-azmanagedcassandracluster
+schema: 2.0.0
+---
+
+# Get-AzManagedCassandraCluster
+
+## SYNOPSIS
+Gets the ManagedCassandra Clusters.
+
+## SYNTAX
+
+```
+Get-AzManagedCassandraCluster [-ResourceGroupName <String>] [-ClusterName <String>]
```

Comment:
Please follow the design here https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/949